### PR TITLE
Adding manual guide to rings

### LINKED ISSUE #1075: How do I add and remove rings manually?
**As a:**

SPK User

**I want:**

A document that details how I should create rings manually, while `spk ring add`, and `spk ring delete` are in progress.

**So that:**

I can follow a set of steps to add and remove rings from bedrock.yaml and my build pipelines, and be ready for when `spk` supports these features.

**Describe the solution you'd like:**

- A document (word or markdown) that details the minimum set of steps that are necessary to add/remove rings from my HLD and pipelines

**Acceptance Criteria:**

Document should cover the following scenarios:

- [x] Detail in-progress issues that cover the scenarios that are to be done manually
- [x] Detail the process to add a ring manually
- [x] Detail the process to remove a ring manually

**Describe alternatives you've considered:**

**Additional context:**

While https://github.com/microsoft/bedrock/issues/955 is in progress, we should have a set of manual steps necessary for adding and removing rings so that our users are ready to adopt the `spk` native commands. 

**Does this require updates to documentation?:**

yes

### PR DESCRIPTION
**Pull request by bnookala**
Fixes https://github.com/microsoft/bedrock/issues/1075

### CODE REVIEW

File: guides/manual-guide-to-rings.md [hunk 1]
```diff
@@ -0,0 +1,323 @@
+# A manual guide to utilizing rings
```
Comment (timfpark) [inline guides/manual-guide-to-rings.md]: Consider title casing this "A Manual Guide to Managing Rings"

File: guides/manual-guide-to-rings.md [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+```
+
+A user wants to add a ring `test-new-homepage`, they first ensure that the git
+branch exists:
+
+- `git checkout dev -b test-new-homepage`
+
+and then invoke the relevant `spk` command to add a `ring`:
+
+- `spk ring create test-new-homepage`
```
Comment (mtarng) [inline guides/manual-guide-to-rings.md:76]: `spk ring add`

**Review by yradsmikham — APPROVED** [review]

File: guides/manual-guide-to-rings.md [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+- prod
+- qa
+```
+
+A user wants to add a ring `test-new-homepage`, they first ensure that the git
+branch exists:
+
+- `git checkout dev -b test-new-homepage`
+
+and then invoke the relevant `spk` command to add a `ring`:
```
Comment (mtarng) [inline guides/manual-guide-to-rings.md]: This workflow is a bit confusing. This is suggesting modifying `bedrock.yaml` in this new branch, which it should be done in the branch, usually `master`, that the hld-lifecycle pipeline triggers off of.

**Review by timfpark — APPROVED** [review]

File: guides/manual-guide-to-rings.md (hunk 2 above)
Comment (mtarng) [reply guides/manual-guide-to-rings.md:76]: Nevermind, we're going with `spk ring create`

File: guides/manual-guide-to-rings.md [hunk 4]
```diff
@@ -0,0 +1,323 @@
+# A manual guide to utilizing rings
+
+### Introduction
```
Comment (evanlouie) [inline guides/manual-guide-to-rings.md]: Maybe decrement all the headings by 1. We don't have any `h2`s

**Review by evanlouie — APPROVED** [review]

File: guides/manual-guide-to-rings.md (hunk 4 above)
Comment (bnookala) [reply guides/manual-guide-to-rings.md]: fixed

File: guides/manual-guide-to-rings.md (hunk 1 above)
Comment (bnookala) [reply guides/manual-guide-to-rings.md]: fixed

File: guides/manual-guide-to-rings.md (hunk 3 above)
Comment (bnookala) [reply guides/manual-guide-to-rings.md]: cleared this up

**Pull request merged**